src/pages/Search/SearchTypeMenuNarrow.tsx

```diff
@@ -170,7 +170,7 @@ function SearchTypeMenuNarrow({typeMenuItems, activeItemIndex, queryJSON, title,
                             />
                             <Text
                                 numberOfLines={1}
-                                style={[styles.textStrong, styles.flexShrink1, styles.fontSizeLabel]}
+                                style={[styles.textStrong, styles.flexShrink1, styles.label]}
```

Comment:
Coming from the checklist in #53590, this change does not cover most cases, so we need to update it by adding the `textLineHeightNormal` style based on container style